tensorflow_addons/image/median_filter_2d.py

```diff
@@ -21,7 +21,7 @@
 
 
 @tf.function
-def median_filter_2D(image, filter_shape=(3, 3)):
+def median_filter_2D(image, filter_shape=(3, 3), name=None):
```

Comment:
Consider `tf.nn.conv2d`, `scipy.ndimage.median_filter` and `scipy.ndimage.median_filter2d`, should this be one of `median_filter` or `median_filter2d`?

https://www.tensorflow.org/api_docs/python/tf/nn/conv2d
https://docs.scipy.org/doc/scipy/reference/ndimage.html

Reply:
@Mainak431 are you okay with this change? I think its the right change and we can make it with ease before the 0.3 release

Reply:
Yes. i am ok.

Reply:
@WindQAQ would you like to squeeze this into this PR? I believe we'd prefer `median_filter2d` as there may be a 1d coming in the future

Reply:
Another PR would be better for me. In fact, I suppose we could create a `filters.py` or a `filters` folder to put all these filters into one file/folder if many useful filtering-based operations would be coming up. Maybe I can help this in that separate PR.

Reply:
Sounds perfect. That follows the style seen in keras modules.